regression/goto-instrument/array_01/test.desc

```diff
@@ -0,0 +1,18 @@
+CORE
+main.c
+--dump-c 
+^        signed int \*\*Array;$
+^        void \*return_value_malloc\$1;$
+^        return_value_malloc\$1=malloc\(\(signed int\)\(2ul \* sizeof\(signed int \*\)
```

Comment:
Those type casts to `signed int` aren't portable - you'll need `long` types here (as well) as the failing regression tests indicate.